### PR TITLE
Language block synchronization

### PR DESCRIPTION
**Pull request by Arhell**
issue #20480
/language pl

### CODE REVIEW

File: README-pl.md [hunk 1]
```diff
@@ -22,14 +22,17 @@ Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz
...
-|[README po francusku](README-fr.md)|[README po koreańsku](README-ko.md)|
-|[README po niemiecku](README-de.md)|[README po portugalsku](README-pt.md)|
-|[README w hindi](README-hi.md)|[README po hiszpańsku](README-es.md)|
-|[README po indonezyjsku](README-id.md)|[README po chińsku](README-zh.md)|
-|[README po japońsku](README-ja.md)|[README po polsku](README-pl.md)|
-|||
+
+|                                        |                                        |
+|----------------------------------------|----------------------------------------|
+| [README po angielski](README.md)       | [README po francusku](README-fr.md)    |
```
Comment (kpucynski) [inline README-pl.md:28]:
```suggestion
| [README po angielsku](README.md)       | [README po francusku](README-fr.md)    |
```

File: README-pl.md [hunk 2]
```diff
@@ -22,14 +22,17 @@ Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz
...
+| [README po koreańsku](README-ko.md)    | [README po niemiecku](README-de.md)    |
+| [README po portugalsku](README-pt.md)  | [README po hindi](README-hi.md)        |
+| [README po hiszpańsku](README-es.md)   | [README po indonezyjsku](README-id.md) |
+| [README po chińsku](README-zh.md)      | [README po japońsku](README-ja.md)     |
+| [README po wietnamski](README-vi.md)   | [README po rosyjski](README-ru.md)     |
```
Comment (kpucynski) [inline README-pl.md:33]:
```suggestion
| [README po wietnamsku](README-vi.md)   | [README po rosyjsku](README-ru.md)     |
```

File: README-pl.md [hunk 3]
```diff
@@ -22,14 +22,17 @@ Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz
...
+| [README po włoski](README-it.md)       | [README po ukraiński](README-uk.md)    |
```
Comment (kpucynski) [inline README-pl.md:34]:
```suggestion
| [README po włosku](README-it.md)       | [README po ukraińsku](README-uk.md)    |
```

File: README-pl.md [hunk 4]
```diff
@@ -22,14 +22,17 @@ Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz
...
-|[README w hindi](README-hi.md)|[README po hiszpańsku](README-es.md)|
-|[README po indonezyjsku](README-id.md)|[README po chińsku](README-zh.md)|
-|[README po japońsku](README-ja.md)|[README po polsku](README-pl.md)|
-|||
+
+|                                        |                                        |
+|----------------------------------------|----------------------------------------|
+| [README po angielski](README.md)       | [README po francusku](README-fr.md)    |
+| [README po koreańsku](README-ko.md)    | [README po niemiecku](README-de.md)    |
+| [README po portugalsku](README-pt.md)  | [README po hindi](README-hi.md)        |
```
Comment (kpucynski) [inline README-pl.md:30]:
```suggestion
| [README po portugalsku](README-pt.md)  | [README w hindi](README-hi.md)        |
```

File: README-pl.md (hunk 1 above)
Comment (Arhell) [reply README-pl.md]: corrected

File: README-pl.md (hunk 2 above)
Comment (Arhell) [reply README-pl.md]: corrected

File: README-pl.md (hunk 3 above)
Comment (Arhell) [reply README-pl.md]: corrected

File: README-pl.md (hunk 4 above)
Comment (Arhell) [reply README-pl.md]: corrected

**Comment by Arhell** [comment]
@kpucynski
Thank you!

**Comment by sftim** [comment]
@kpucynski does this look good to you?

**Pull request merged**